CodeConverter/Shared/ConversionOptions.cs

```diff
@@ -6,6 +6,7 @@ namespace ICSharpCode.CodeConverter.Shared
     public class ConversionOptions
     {
         public string RootNamespaceOverride { get; set; }
+        public object LanguageVersionOverride { get; set; }
```

Comment:
Is it possible to just set the `TargetCompilationOptionsOverride.ParseOptions.LanguageVersion` instead of adding a new property?

Reply:
Yes, it is. I have get confused with these options.